src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+  --pf-c-sequence__link--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-sequence__link--BorderBottomColor: var(--pf-global--Color--200);
+  --pf-c-sequence__link--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__link--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--active--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__link--focus--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Number
+  --pf-c-sequence__number--Padding: var(--pf-global--spacer--xs);
+  --pf-c-sequence__number--MarginRight: var(--pf-global--spacer--lg);
+  --pf-c-sequence__number--MinWidth: var(--pf-global--spacer--lg);
+  --pf-c-sequence__number--MaxHeight: var(--pf-global--spacer--lg);
+  --pf-c-sequence__number--hover--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--hover--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--hover--MaxHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--active--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--active--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--active--MaxHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--focus--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-sequence__number--focus--MinWidth: var(--pf-global--spacer--xl);
+  --pf-c-sequence__number--focus--MaxHeight: var(--pf-global--spacer--xl);
+
+  // Number Color
+  --pf-c-sequence__number--Color: var(--pf-global--Color--200);
+  --pf-c-sequence__number--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--active--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--focus--Color: var(--pf-global--primary-color--100);
+
+  // Number Font
+  --pf-c-sequence__number--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-sequence__number--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-sequence__number--hover--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--active--FontSize: var(--pf-global--FontSize--xl);
+  --pf-c-sequence__number--focus--FontSize: var(--pf-global--FontSize--xl);
+
+  // Number Border
+  --pf-c-sequence__number--BorderColor: var(--pf-global--Color--200);
+  --pf-c-sequence__number--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-sequence__number--BorderWeight: var(--pf-global--BorderWidth--md);
+  --pf-c-sequence__number--hover--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--active--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__number--focus--BorderColor: var(--pf-global--primary-color--100);
+
+  // Content
+  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--xl);
+
+  // content Title
+  --pf-c-sequence__content--title--MaxWidth: 100px;
+
+  // Content Title Color
+  --pf-c-sequence__content-title--Color: var(--pf-global--Color--light-100);
```

Comment:
Per the design, shouldn't this match the circle/bottom-border color? I also can't tell for certain in the PDF, but it looks like the text and circle are semi-transparent, too (like the bottom border)

Reply:
In the sketch file it looks like the title color is white, and the status color is different to the number color. The number color is a solid, but you are right the border color is transparent so will change that